editor.planx.uk/src/pages/FlowEditor/components/EditHistory.tsx

```diff
@@ -150,18 +178,19 @@ const EditHistory = () => {
                     </Typography>
                   </Box>
                   {i > 0 && op.actor && canUserEditTeam(teamSlug) && (
-                    <IconButton
-                      title="Restore to this point"
-                      aria-label="Restore to this point"
-                      onClick={() => handleUndo(i)}
-                      onMouseEnter={() => setFocusedOpIndex(i)}
-                      onMouseLeave={() => setFocusedOpIndex(undefined)}
-                    >
-                      <RestoreOutlined
-                        fontSize="large"
-                        color={inUndoScope(i) ? "inherit" : "primary"}
-                      />
-                    </IconButton>
+                    <TooltipWrap title="Restore to this point">
```

Comment:
aha this tooltip is much more prominent than IconButton `title` :bulb: good call!